packages/fiori/src/ShellBar.ts

```diff
@@ -436,14 +395,20 @@ class ShellBar extends UI5Element {
 	@property({ type: Boolean, noAttribute: true })
 	_fullWidthSearch!: boolean;
 
-	@property({ type: Boolean, noAttribute: true })
-	_coPilotPressed!: boolean;
-
 	@property({ type: Boolean, noAttribute: true })
 	_isXXLBreakpoint!: boolean;
 
 	/**
-	 * Defines the `ui5-shellbar` aditional items.
+	 * Defines the `ui5-shellbar` assistant.
```

Comment:
Best practice is to not mention the component name inside the description or property since if the class is extended the description will be used for the subclass.